README.md

```diff
@@ -128,6 +128,19 @@ Follow our [<img src='https://cdn2.iconfinder.com/data/icons/social-media-2285/5
 
 关注我们 [<img src='https://www.iconfinder.com/icons/5060515/download/svg/512' width='20px' /> 知乎专栏](https://zhuanlan.zhihu.com/c_1255493231133417472) 获取DJL最新的内容！
 
+## Useful Links
+
+* [DJL Website](https://djl.ai/)
+* [Documentation](https://docs.djl.ai/)
+* [DJL Demos](https://docs.djl.ai/docs/demos/index.html)
+* [Dive into Deep Learning Book Java version](https://d2l.djl.ai/)
+
+Also checkout these cool projects using DJL:
```

Comment:
Maybe add these to the index on the demo page? It seems a bit hard to maintain a list of other repos in our main readme.

Reply:
done